### ISSUE TITLE
Request: Tutorial or example code for using Tone.js with AudioWorklet

### ISSUE THREAD

**Issue by natschager** [Tonejs/Tone.js#1180]
Hello all,

I'm trying to learn WebAudio stuff recently, and am enjoying tone.js + p5.js together. However, I am also bumping up against a performance limit with the polysynth on some browsers (lots of objects making noises), and figured one solution would be putting the audio onto audio worklets. I can't find any documentation or examples of how to do this with Tone.js, except that this function exists: createAudioWorkletNode and the resulting class exists. Guessing it is also a gap in my WebAudio knowledge, but I've actually been using Tone as my entry point for WebAudio and am not used to working from WebAudio back to Tone. 

Basically, I was wondering if there were any example projects you could point me to for how to use the function/class above, and also wanted to suggest adding a page to the GitHub Wiki about using audio worklet with the library.

Thanks for your time and best,
-Nat
